### PR TITLE
Cosmetic update - docker-compose + readme

### PR DESCRIPTION
**Pull request by kamil-p**
Fixed version of services. Improved README. Added gitignore.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,30 +2,30 @@ version: '2'
...
       PGDATA: /data/postgres
     volumes:
-       - /data/postgres:/data/postgres
+    - /data/postgres:/data/postgres
     ports:
-      - "5432:5432"
+    - "5432:5432"
     networks:
-      - postgres
-    restart: unless-stopped
```
Comment (khezen) [inline docker-compose.yml]: please keep `unless-stopped`

**Comment by khezen** [comment]
Hi Kamil, 

The docker-compose is not intended for production use but simply a convenient way to run pgadmin. Therefore I prefer to keep untagged images as default instead of fixed version. 

I don't understand the content of the `.gitignore`. Maybe you can explain that part.

**Comment by kamil-p** [comment]
> Hi Kamil,
> 
> The docker-compose is not intended for production use but simply a convenient way to run pgadmin. Therefore I prefer to keep untagged images as default instead of fixed version.
> 
> I don't understand the content of the `.gitignore`. Maybe you can explain that part.

Hi, First of all, I would like to thank you for your brilliant repository it helped me a lot. 

I used tagged versions because when something changes it could crush a docker build. Without it may be hard to find which versions were the lates working. 

Answering your question about ".gitignore" i is for ignoring ".idea" which is jetbrains IDEs (intellij, phpstorm, pycharm, webstorm) project settings also added "/data" to the ignore because after first build "data" directory appears in the project which represents a volume without it "git status" command shows files not added to the repo.

**Comment by khezen** [comment]
I don't think turning a single docker-compose file into IDE project is the right thing to do. `/data` is an absolute path  and shouldn't create a folder in your project.

**Pull request closed without merging**